pkg/controllers/provisioning/scheduling/suite_test.go

```diff
@@ -4597,6 +4597,115 @@ var _ = Context("Scheduling", func() {
 			}
 		})
 	})
+
+	Describe("Dynamic Resource Allocation (DRA)", func() {
+		It("should not create NodeClaims for DRA pods and return DRA error", func() {
+			nodePool := test.NodePool()
+			ExpectApplied(ctx, env.Client, nodePool)
+
+			// Create a pod with DRA requirements (container consumes ResourceClaims)
+			DRAEnabledPod := &corev1.Pod{
```

Comment:
nit: variable naming should be camel case
```suggestion
			draEnabledPod := &corev1.Pod{
```